boards/p-nucleo-wb55/include/periph_conf.h

```diff
@@ -21,71 +21,37 @@
 #define PERIPH_CONF_H
 
 #include "periph_cpu.h"
-#include "cfg_i2c1_pb8_pb9.h"
-#include "cfg_rtt_default.h"
-#include "cfg_timer_tim2.h"
 
-#ifdef __cplusplus
-extern "C" {
-#endif
+/* Add specific clock configuration (HSE, LSE) for this board here */
+#define CLOCK_CORECLOCK_MAX             MHZ(64)
 
-/**
- * @name    Clock system configuration
- * @{
- */
-/* 0: no external high speed crystal available
- * else: actual crystal frequency [in Hz] */
-#define CLOCK_HSE           (32000000U)
+#ifndef CONFIG_BOARD_HAS_LSE
+#define CONFIG_BOARD_HAS_LSE            1
+#endif
 
-#ifndef CLOCK_LSE
-/* 0: no external low speed crystal available,
- * 1: external crystal available (always 32.768kHz)
- */
-#define CLOCK_LSE           (1)
+#ifndef CONFIG_BOARD_HAS_HSE
+#define CONFIG_BOARD_HAS_HSE            1
 #endif
 
-/* 0: enable MSI only if HSE isn't available
- * 1: always enable MSI (e.g. if USB or RNG is used)*/
-#define CLOCK_MSI_ENABLE    (1)
+#define CLOCK_HSE                       MHZ(32)
 
-#ifndef CLOCK_MSI_LSE_PLL
-/* 0: disable Hardware auto calibration with LSE
- * 1: enable Hardware auto calibration with LSE (PLL-mode)
- */
-#define CLOCK_MSI_LSE_PLL   (1)
+#ifndef CONFIG_CLOCK_PLL_M
+#define CONFIG_CLOCK_PLL_M              (5)
 #endif
 
-/* give the target core clock (HCLK) frequency [in Hz], maximum: 64MHz */
-#define CLOCK_CORECLOCK     (64000000U)
-/* PLL configuration: make sure your values are legit!
- *
- * compute by: CORECLOCK = (((PLL_IN / M) * N) / R)
- * with:
- * PLL_IN:  input clock, HSE or MSI @ 48MHz
- * M:       pre-divider,  allowed range: [1:8]
- * N:       multiplier,   allowed range: [8:86]
- * R:       post-divider, allowed range: [2:8]
- *
- * Also the following constraints need to be met:
- * (PLL_IN / M)     -> [4MHz:16MHz]
- * (PLL_IN / M) * N -> [64MHz:344MHz]
- * CORECLOCK        -> 64MHz MAX!
- */
```

Comment:
This kind of information is completely lost with your changes.

Reply:
I readded it in the `cfg_clock_default.h` header.